monitoring.sh

```diff
@@ -38,22 +39,11 @@ if [[ $ALIBUILD_O2_TESTS ]]; then
   make test
 fi
 
+
 #ModuleFile
 mkdir -p etc/modulefiles
-cat > etc/modulefiles/$PKGNAME <<EoF
-#%Module1.0
-proc ModulesHelp { } {
-  global version
-  puts stderr "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
-}
-set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
-module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
-# Dependencies
-module load BASE/1.0                                                                                \\
-            ${BOOST_REVISION:+boost/$BOOST_VERSION-$BOOST_REVISION}                                 \\
-            ${APMON_CPP_REVISION:+ApMon-CPP/$APMON_CPP_VERSION-$APMON_CPP_REVISION}                 \\
-            ${GCC_TOOLCHAIN_REVISION:+GCC-Toolchain/$GCC_TOOLCHAIN_VERSION-$GCC_TOOLCHAIN_REVISION}
-
+alibuild-generate-module > etc/modulefiles/$PKGNAME
```

Comment:
As a note, you can use `alibuild-generate-module --bin --lib` and it will generate the LD_LIBRARY_PATH and PATH automatically. It will not however export the `MONITORING_ROOT`, which you will have to do by hand.

Reply:
Ok, cool, I'll do that. On the other hand I think no1 is using the env var anymore so I'll just drop it.